### PR TITLE
Minor fixes to test_Affy.py

### PR DESCRIPTION
**Pull request by jvfe**
This pull request addresses issue #2835 and #2552 

- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

I've substituted the assert statement on line 39 for assertGreater() as specified [here](https://github.com/biopython/biopython/issues/2835#issuecomment-626322663). Also applied black styling to the file as specified by #2552

### CODE REVIEW

File: Tests/test_Affy.py [hunk 1]
```diff
@@ -82,75 +86,99 @@ def testAffy3(self):
 
             global message
             try:
-                numpy.testing.assert_allclose(record.intensities,
-                                              [[234.0, 170.0, 22177.0, 164.0, 22104.0],
-                                               [188.0, 188.0, 21871.0, 168.0, 21883.0],
-                                               [188.0, 193.0, 21455.0, 198.0, 21300.0],
-                                               [188.0, 182.0, 21438.0, 188.0, 20945.0],
-                                               [193.0, 20370.0, 174.0, 20605.0, 168.0]])
+                numpy.testing.assert_allclose(
```
Comment (jvfe) [inline Tests/test_Affy.py:89]: I don't know how you're handling the numpy matrices when applying black styling. If preferred, I can make another commit turning fmt: off for the matrices.

Comment (peterjc) [reply Tests/test_Affy.py:89]: These look fine - we do have concerns when the layout hides some meaning. Specifically small integer matrices can get squashed into one line, when there are much better shown one line per row.

Comment (peterjc) [reply Tests/test_Affy.py:89]: See discussion on #3089 about this and https://github.com/biopython/biopython/pull/3089/commits/cb065f6ffe3f13bf988df5a4fb07f99793a4547f

Comment (jvfe) [reply Tests/test_Affy.py:89]: Ok! Thanks for clarifying.

**Comment by peterjc** [comment]
And #3070 strikes again, but in triplicate:

```
======================================================================
ERROR: test_transfer (common_BioSQL.InDepthLoadTest)
Make sure can load record into another namespace.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\biopython\Tests\common_BioSQL.py", line 957, in setUp
    driver=DBDRIVER, user=DBUSER, passwd=DBPASSWD, host=DBHOST, db=TESTDB
  File "C:\projects\biopython\build\lib.win-amd64-3.7\BioSQL\BioSeqDatabase.py", line 110, in open_database
    conn = connect(**kw)
  File "C:\Py\lib\site-packages\psycopg2\__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
======================================================================
ERROR: test_addition (common_BioSQL.SeqInterfaceTest)
Check can add DBSeq objects together.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\biopython\Tests\common_BioSQL.py", line 440, in setUp
    load_database("GenBank/cor6_6.gb")
  File "C:\projects\biopython\Tests\common_BioSQL.py", line 246, in load_database
    TESTDB = create_database()
  File "C:\projects\biopython\Tests\common_BioSQL.py", line 216, in create_database
    _do_db_cleanup()
  File "C:\projects\biopython\Tests\common_BioSQL.py", line 145, in _do_db_cleanup
    driver=DBDRIVER, host=DBHOST, user=DBUSER, passwd=DBPASSWD, db=TESTDB
  File "C:\projects\biopython\build\lib.win-amd64-3.7\BioSQL\BioSeqDatabase.py", line 110, in open_database
    conn = connect(**kw)
  File "C:\Py\lib\site-packages\psycopg2\__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
======================================================================
ERROR: test_seq (common_BioSQL.SeqInterfaceTest)
Make sure Seqs from BioSQL implement the right interface.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\biopython\Tests\common_BioSQL.py", line 443, in setUp
    driver=DBDRIVER, user=DBUSER, passwd=DBPASSWD, host=DBHOST, db=TESTDB
  File "C:\projects\biopython\build\lib.win-amd64-3.7\BioSQL\BioSeqDatabase.py", line 110, in open_database
    conn = connect(**kw)
  File "C:\Py\lib\site-packages\psycopg2\__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
----------------------------------------------------------------------
Ran 519 tests in 828.684 seconds
FAILED (failures = 1)
```

I will restart AppVeyor.

**Pull request merged**